### PR TITLE
Integrate all-contributors

### PR DESCRIPTION
**Pull request by bmullan91**
Includes:
- Two new npm scripts that alias `all-contributors` commands
- `.all-contributorsrc`
- Updated `CONTRIBUTING.md` with info on using `all-contributors`
- Updates `README.md` adding all contributors 🙌

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "prettier": "pretty-quick"
+    "prettier": "pretty-quick",
+    "add-contrib": "all-contributors add",
+    "generate-contribs": "all-contributors generate"
```
Comment (jackdclark) [inline package.json]: For consistency with other tasks these could be `contributors:add` and `contributors:generate`, what do you think?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 [![Coverage Status](https://coveralls.io/repos/github/nearform/graphql-hooks/badge.svg?branch=master)](https://coveralls.io/github/nearform/graphql-hooks?branch=master)
 ![](https://img.shields.io/bundlephobia/minzip/graphql-hooks.svg?style=flat)
 [![npm version](https://badge.fury.io/js/graphql-hooks.svg)](https://badge.fury.io/js/graphql-hooks)
+[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors)
```
Comment (jackdclark) [inline README.md]: This is hardcoded to 4, is it updated by the bot?

File: package.json (hunk 1 above)
Comment (bmullan91) [reply package.json]: Yea I was trying to shorten it - I'll stick to consistency 👍

File: README.md (hunk 2 above)
Comment (bmullan91) [reply README.md]: It's updated by the bot / npm commands automatically 🙌

**Pull request merged**